system-test/spanner.ts

```diff
@@ -933,6 +933,25 @@ describe('Spanner', () => {
           })
         );
     });
+
```

Comment:
Sorry if you're still working on this, but it would be good to have some unit tests as it would be checked by the code cov tool.

Reply:
I've added a couple of unit tests.